board/miyoo/main/apps/usb-hid/usb-hid.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/busybox sh
+if [ `cat /sys/devices/platform/soc/1c13000.usb/musb-hdrc.1.auto/mode` != "b_peripheral" ]; then
+	echo -e "\e[31mFirst connect handheld to device!\e[0m"&>$(tty)
+	sleep 2
+	exit
+fi
+gadget-vid-pid-remove 0x1d6b:0x0104
+gadget-hid
+echo -e "\e[32mStarting USB HID mode...\e[0m"&>$(tty)
```

Comment:
I did not know you can display msgs in tty that way. Great!

Reply:
The TTY isn't always displaying a passed msg that way -> try for e.g. running "File System Check" app and then see if USB-HID is showing any stuff on console. There are also some random situations when even blinking cursor dissapears.

Reply:
Blinking cursor disappears, because main script set black colour for font, I will check